### ISSUE TITLE
ConnectedComponentsNepheleITCase fails on Travis

### ISSUE THREAD

**Issue by rmetzger** [stratosphere/stratosphere#338]
```
Running eu.stratosphere.pact.test.iterative.nephele.ConnectedComponentsNepheleITCase Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.124 sec <<< FAILURE!
```

also eu.stratosphere.pact.test.pactPrograms.WordCountUnionReduceITCase
see https://travis-ci.org/stratosphere/stratosphere/jobs/15221965
https://travis-ci.org/stratosphere/stratosphere/jobs/15066210
https://travis-ci.org/stratosphere/stratosphere/jobs/15298533
https://travis-ci.org/stratosphere/stratosphere/jobs/15298534

I'm already looking into maven to see why it accepts errored tests (I though this was already fixed)

**Comment by StephanEwen** [comment]
I noticed that as well and restructured the test. The Problem is not that the tests themselves fail, but the class has many static subclasses with tests, which is not properly handled.

**Comment by StephanEwen** [comment]
Fixed in cfad3eb34595ee7efaf8f818f4bd8561a50838fd
